params/config_arbitrum.go

```diff
@@ -29,6 +29,8 @@ type ArbitrumChainParams struct {
 	InitialArbOSVersion       uint64
 	InitialChainOwner         common.Address
 	GenesisBlockNum           uint64
+	MaxCodeSize               uint64 // Maximum bytecode to permit for a contract. 0 value implies params.MaxCodeSize
+	MaxInitCodeSize           uint64 // Maximum initcode to permit in a creation transaction and create instructions. o value implies params.MaxInitCodeSize
```

Comment:
Let's add `json:"omitempty"` here as to not change the serialization of existing chain configs